buildme

```diff
@@ -25,7 +25,7 @@ if [ $ARCH = "armv6l" ] || [ $ARCH = "armv7l" ] || [ $ARCH = "aarch64" ]; then
 	if [ $ARCH = "armv6l" ]; then
 		make
 	else
-		make -j4
+		make -j4 VERBOSE=True
```

Comment:
This results in logging of the exact commands used to build the code, which was very useful when we were trying to figure out where to insert `wiringPi` to get the library actually linked and included!

I could take this out, but since we'll probably build this package so infrequently, and might run into annoying build issues again, I was thinking I might as well leave it in for our future selves.

Reply:
I vote we keep it.

Reply:
👍 to leaving it in